Makefile

```diff
@@ -167,3 +168,6 @@ scan:
 		--severity="HIGH,CRITICAL" \
 		--ignore-unfixed \
 		./
+
+print-%:
+	@echo -n $($*)
```

Comment:
essentially using this to print out a value we have set in the Makefile. lmk if there's a preferable way to do this. ex:
```
make print-PREVIOUS_K0S_VERSION
```

Reply:
this actually makes more sense to me